internal/explorer.point/src/components/LinkPointToSol.jsx

```diff
@@ -0,0 +1,119 @@
+import { useState, useEffect } from 'react';
+import Swal from 'sweetalert2';
+import { useAppContext } from '../context/AppContext';
+
+const USER_REJECTION_CODE = 4001;
+const LS_KEY = 'point_to_sol_msg';
+const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+const styles = {
+    container: {
+        borderRadius: 4,
+        boxShadow: '0px 7px 15px 0 rgba(0, 0, 0, .15)',
+        padding: 16,
+        marginTop: 24,
+        color: 'rgb(124, 124, 124)',
+        display: 'grid',
+        gridTemplateColumns: '1fr 200px',
+        alignItems: 'center',
+        gridColumnGap: 42,
+    },
+    description: {
+        margin: 0,
+    },
+};
```

Comment:
I kept everything in this new file, including some styles, because I understand there's a big refactor of Point Explorer in progress, so this way I can avoid conflicts and adjust to whatever new design is implemented.